consumer.go

```diff
@@ -83,6 +83,17 @@ func (c *Consumer) Run(ctx context.Context) {
 	c.log.Println("Bye")
 }
 
+// dropInvalidOffsets drops any Offset of type OffsetInvalid from the given OffsetsCommitted argument.
+func dropInvalidOffsets(offsets rdkafka.OffsetsCommitted) rdkafka.OffsetsCommitted {
+	trimmedParts := []rdkafka.TopicPartition{}
```

Comment:
This is actually the information you keep and present, so maybe we should name this `validParts`? or `updatedParts` even?

Reply:
Also how about renaming the method to `filterValidOffsets` or `keepValidOffsets`? Just a suggestion!

Reply:
+1 for "filter<blabla>"

Reply:
Right! This variable should be renamed!